New in June (#2050)
ezsystems/developer-documentation: docs/resources/new_in_doc.md

This page contains recent highlights and notable changes in Ibexa DXP documentation.

## June 2023

### Personalization

> - [Email triggers in User Guide](https://doc.ibexa.co/projects/userguide/en/master/personalization/triggers/)

Reviewer: ```suggestion
- [Email triggers](https://doc.ibexa.co/en/master/personalization/integrate_recommendation_service/#send-emails-with-recommendations)
- [Email triggers in user documentation](https://doc.ibexa.co/projects/userguide/en/master/personalization/triggers/)
```
We have two ways of referring to the user guide, with "user documentation" prevailing. I suggest we decide on one and use it consistently.

### Search

> - [Updated search engines documentation](https://doc.ibexa.co/en/master/search/search_engines/):

Reviewer: ```suggestion
- [Updated search engines documentation](https://doc.ibexa.co/en/master/search/search_engines/search_engines/):
```

    - [Elasticsearch search engine](https://doc.ibexa.co/en/master/search/search_engines/elastic_search/elastic_search_overview/)
    - [Solr search engine](https://doc.ibexa.co/en/master/search/search_engines/solr_search_engine/solr_overview/)
    - [Legacy search engine](https://doc.ibexa.co/en/master/search/search_engines/legacy_search_engine/legacy_search_overview/#legacy-search-engine)

### Commerce

> - [Shipping methods management](https://doc.ibexa.co/projects/userguide/en/master/commerce/shipping_management/work_with_shipping_methods/)
> - [Payment methods management](https://doc.ibexa.co/projects/userguide/en/master/commerce/payment/work_with_payments/)

Reviewer: ```suggestion
- [Shipping methods management in user documentation](https://doc.ibexa.co/projects/userguide/en/master/commerce/shipping_management/work_with_shipping_methods/)
- [Payment methods management in user documentation](https://doc.ibexa.co/projects/userguide/en/master/commerce/payment/work_with_payments/)
```

For the sake of consistency with previous occurrence. It could be "User Guide" if you decide so.
